os_net_config/tests/test_impl_nmstate.py

```diff
@@ -309,6 +309,9 @@ def get_nmstate_ethtool_opts(self, name):
             self.provider.interface_data[name][Ethernet.CONFIG_SUBTREE]
         data[Ethtool.CONFIG_SUBTREE] = \
             self.provider.interface_data[name][Ethtool.CONFIG_SUBTREE]
+        if 'dispatch' in self.provider.interface_data[name].keys():
```

Comment:
This could have been handled with a shorter key than "dispatch" like "dp" or "sh", but I support the use of the \ continuation for readability. 

Another thing you can do to fit things on one line is to assign the base object to a handle, like:

pvdr = self.provider

Then you can use:

pvdr.interface_data[...